traitsui/testing/tester/ui_wrapper.py

```diff
@@ -73,6 +74,57 @@ def __init__(self, target, registries, delay=0):
         self._registries = registries
         self.delay = delay
 
+    def help(self):
+        """ Print help messages.
+        (This function is intended for interactive use.)
+        """
+        # mapping from interaction types to their documentation
+        interaction_to_doc = dict()
+
+        # mapping from location types to their documentation
+        location_to_doc = dict()
+
+        # Let higher priority registry override the documentation from lower
+        # priority registry.
+        for registry in self._registries[::-1]:
```

Comment:
Just for my own understanding, the concept of priority is needed because two registries may have handlers or solvers registered for the same `type_` below.  So, by going in reverse order we override documentation until we have the docs for the highest priority registry?  This would then be the registry used if someone asks for an interaction or location with type_

Reply:
That's right!